### ISSUE TITLE
Cannot read property 'bindAssignmentAtPath' of null

### ISSUE THREAD

**Issue by HitomiTenshi** [rollup/rollup#1702]
Using the latest version `0.51.0` I get the following error during rollup:

```
main-aot.js → dist-app/app.bundle.js...

Conflicting namespaces: node_modules\@swimlane\ngx-charts\release\pie-chart\index.js re-exports 'PieChartComponent' from both node_modules\@swimlane\ngx-charts\release\pie-chart\pie-chart.module.js and node_modules\@swimlane\ngx-charts\release\pie-chart\pie-chart.component.js (will be ignored)
Conflicting namespaces: node_modules\@swimlane\ngx-charts\release\pie-chart\index.js re-exports 'PieArcComponent' from both node_modules\@swimlane\ngx-charts\release\pie-chart\pie-chart.module.js and node_modules\@swimlane\ngx-charts\release\pie-chart\pie-arc.component.js (will be ignored)
Conflicting namespaces: node_modules\@swimlane\ngx-charts\release\pie-chart\index.js re-exports 'PieGridComponent' from both node_modules\@swimlane\ngx-charts\release\pie-chart\pie-chart.module.js and node_modules\@swimlane\ngx-charts\release\pie-chart\pie-grid.component.js (will be ignored)
Conflicting namespaces: node_modules\@swimlane\ngx-charts\release\pie-chart\index.js re-exports 'PieSeriesComponent' from both node_modules\@swimlane\ngx-charts\release\pie-chart\pie-chart.module.js and node_modules\@swimlane\ngx-charts\release\pie-chart\pie-series.component.js (will be ignored)
Conflicting namespaces: node_modules\@swimlane\ngx-charts\release\pie-chart\index.js re-exports 'PieLabelComponent' from both node_modules\@swimlane\ngx-charts\release\pie-chart\pie-chart.module.js and node_modules\@swimlane\ngx-charts\release\pie-chart\pie-label.component.js (will be ignored)
Conflicting namespaces: node_modules\@swimlane\ngx-charts\release\pie-chart\index.js re-exports 'PieSeriesComponent' from both node_modules\@swimlane\ngx-charts\release\pie-chart\pie-chart.module.js and node_modules\@swimlane\ngx-charts\release\pie-chart\pie-series.component.js (will be ignored)
Conflicting namespaces: node_modules\@swimlane\ngx-charts\release\line-chart\index.js re-exports 'LineChartComponent' from both node_modules\@swimlane\ngx-charts\release\line-chart\line-chart.module.js and node_modules\@swimlane\ngx-charts\release\line-chart\line-chart.component.js (will be ignored)
Conflicting namespaces: node_modules\@swimlane\ngx-charts\release\line-chart\index.js re-exports 'LineComponent' from both node_modules\@swimlane\ngx-charts\release\line-chart\line-chart.module.js and node_modules\@swimlane\ngx-charts\release\line-chart\line.component.js (will be ignored)
Conflicting namespaces: node_modules\@swimlane\ngx-charts\release\line-chart\index.js re-exports 'LineSeriesComponent' from both node_modules\@swimlane\ngx-charts\release\line-chart\line-chart.module.js and node_modules\@swimlane\ngx-charts\release\line-chart\line-series.component.js (will be ignored)

[!] TypeError: Cannot read property 'bindAssignmentAtPath' of null
TypeError: Cannot read property 'bindAssignmentAtPath' of null
    at node (C:\Users\johann.rekowski\Documents\Apps\Media-Presenter\node_modules\rollup\dist\rollup.js:11910:9)
    at Set.forEach (<anonymous>)
    at FunctionScope.forEachReturnExpressionWhenCalled (C:\Users\johann.rekowski\Documents\Apps\Media-Presenter\node_modules\rollup\dist\rollup.js:11630:27)
    at Node.forEachReturnExpressionWhenCalledAtPath (C:\Users\johann.rekowski\Documents\Apps\Media-Presenter\node_modules\rollup\dist\rollup.js:12506:17)
    at boundExpressions.forEachAtPath (C:\Users\johann.rekowski\Documents\Apps\Media-Presenter\node_modules\rollup\dist\rollup.js:11370:6)
    at _assignments.get.forEach.assignment (C:\Users\johann.rekowski\Documents\Apps\Media-Presenter\node_modules\rollup\dist\rollup.js:11255:59)
    at Set.forEach (<anonymous>)
    at VariableShapeTracker.forEachAtPath (C:\Users\johann.rekowski\Documents\Apps\Media-Presenter\node_modules\rollup\dist\rollup.js:11255:36)
    at LocalVariable.forEachReturnExpressionWhenCalledAtPath (C:\Users\johann.rekowski\Documents\Apps\Media-Presenter\node_modules\rollup\dist\rollup.js:11368:25)
    at Node.forEachReturnExpressionWhenCalledAtPath (C:\Users\johann.rekowski\Documents\Apps\Media-Presenter\node_modules\rollup\dist\rollup.js:12637:20)
```

This didn't happen with previous versions.

My config:

```
import builtins from 'rollup-plugin-node-builtins';
import nodeResolve from 'rollup-plugin-node-resolve';
import commonjs from 'rollup-plugin-commonjs';

export default {
  input: 'main-aot.js',
  output: {
    file: 'dist-app/app.bundle.js',
    format: 'iife',
    sourcemap: false
  },
  onwarn: (warning: any) => {
    // Skip certain warnings
    if (warning.code === 'THIS_IS_UNDEFINED') return;

    // console.warn everything else
    console.warn(warning.message);
  },
  plugins: [
    builtins(),
    nodeResolve({
      module: true,
      jsnext: true,
      main: true,
      browser: true
    }),
    commonjs({
      include: [
        'node_modules/**'
      ],
      sourceMap: false,
      namedExports: {
        'node_modules/bcryptjs/dist/bcrypt.js': [
          'compare',
          'hash'
        ],
        'node_modules/xlsx/xlsx.js': [
          'utils',
          'write'
        ]
      }
    })
  ]
}
```

**Comment by lukastaegert** [comment]
Hi @HitomiTenshi,
thank you for filing this issue! I think I already know what the issue is, namely I forgot to handle empty return statements. Shouldn't be a big issue, I will probably be able to publish a fix until tomorrow. Until then, please stick with 0.50.1.

**Comment by lukastaegert** [comment]
Actually I was a little faster, should be fixed in 0.51.1
